docs/extensions_dev.md

```diff
@@ -177,10 +177,7 @@ document can then be saved by the user in the usual manner.
 ## Themes
 A theme is a JupyterLab extension that uses a `ThemeManager` and can be
 loaded and unloaded dynamically.  The package must include all static assets
-that are referenced by `url()` in its CSS files.  The `url()` paths in a CSS
-file served by the Jupyter server must start with the path
-`'./lab/api/themes/<foo>/'`, where `foo` is the name of the
-package. Note that `'@import'` paths are still given as relative paths, e.g. (`'@import './foo.css';`).
+that are referenced by `url()` in its CSS files.
```

Comment:
do we need to mention that these URLs need to be relative to the location of the referring CSS file in the theme src directory?

Reply:
Updated the docs in b034509, sound good?